fog/ledger/server/src/key_image_service.rs

```diff
@@ -107,6 +108,9 @@ impl<E: LedgerEnclaveProxy> KeyImageService<E> {
         &mut self,
         request: attest::NonceMessage,
     ) -> Result<attest::NonceMessage, EnclaveError> {
+        let tracer = mc_util_telemetry::tracer!();
+        let _guard =
+            Context::current_with_span(tracer.start("check_key_image_store_auth")).attach();
```

Comment:
Other places in mobilecoin used the `in_span()` method, but wanted to avoid unnecessary nesting so just use the guard version

FYI the implementation of `in_span()`
```rust
        let span = self.start(name);
        let cx = Context::current_with_span(span);
        let _guard = cx.clone().attach();
        f(cx)
```